src/builder.ts

```diff
@@ -61,6 +61,11 @@ export class UnknownQuery extends EntityQuery<z.ZodUnknown> {
       schema: z.array(z.unknown()),
     });
   }
+
+  deref() {
+    this.query += "->";
```

Comment:
do we need to do any enforcement that this should be the last in the chain?

Reply:
Generally, a `.grab` call will be last in the chain, and having `deref` as last in the chain would leave you with an `unknown` result. So having `deref` last in the chain doesn't make much sense from a TS perspective, but groq doesn't care if you do it.